elements/lisk-transactions/src/create_sendable.ts

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import { validateAddress, validatePublicKey } from '@liskhq/lisk-validator';
+
+import { TransactionJSON } from './transaction_types';
+
+import {
+	getAddressFromPublicKey,
+	getPrivateAndPublicKeyBytesFromPassphrase,
+} from '@liskhq/lisk-cryptography';
+
+const validateRequiredInputs = (
```

Comment:
maybe `export` this for testing, and maybe rename to `validateSendableInputs`?

Reply:
Prefer sth with required :)

Reply:
I changed to "obligatory"